src/state/airdrop/reducer.ts

```diff
@@ -3,11 +3,17 @@ import { AirdropAction, AirdropActionType } from './actions';
 export interface AirdropState {
   thankYouSeen: boolean;
   thankYouFlowCompleted: boolean;
+  isLoading: boolean;
+  error: boolean;
```

Comment:
the error should be string, catch errors message from backend, sagas, etc.

Reply:
we agreed we don't care for messages here, we have only one screen for all errors, no point in storing them, right?

Reply:
doesn't matter if we care or not, maybe we could in the future, that's why we should gather the info about errors. you can always make a selector hasError(state => !!state.error) for parsing empty string to boolean

Reply:
bump2;p

Reply:
bump3

Reply:
done, done2, done3 :D sorry, misunderstanding